src/subdomains/core/aml/aml-helper.service.ts

```diff
@@ -66,8 +66,6 @@ export class AmlHelperService {
       // crypto input
       if (entity.cryptoInput.amlCheck !== CheckStatus.PASS) errors.push(AmlError.INPUT_AML_CHECK_FAILED);
       if (!entity.cryptoInput.isConfirmed) errors.push(AmlError.INPUT_NOT_CONFIRMED);
-      if (entity instanceof BuyCrypto && !entity.userData.cryptoCryptoAllowed)
-        errors.push(AmlError.CRYPTO_CRYPTO_NOT_ALLOWED);
```

Comment:
Remove `cryptoCryptoAllowed` on UserData? Or maybe in a second step?

Reply:
I would do it in another step

Reply:
I made a ticket for me for it 👍